src/librustdoc/clean/types.rs

```diff
@@ -1014,14 +991,85 @@ pub(crate) fn hir_attr_lists<'a, I: IntoIterator<Item = &'a hir::Attribute>>(
         .flatten()
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct CfgInfo {
+    hidden_cfg: FxHashSet<Cfg>,
+    current_cfg: Cfg,
+    doc_auto_cfg_active: bool,
```

Comment:
nit: maybe call this just `auto_cfg_active`

Reply:
Done!